apitools/base/py/transfer.py

```diff
@@ -632,7 +631,7 @@ def __ConfigureMultipartRequest(self, http_request):
 
     # encode the body: note that we can't use `as_string`, because
     # it plays games with `From ` lines.
-    fp = StringIO.StringIO()
+    fp = six.StringIO()  # email: cStringIO in Py2 / io.StringIO in Py3.
```

Comment:
Yeah the comment is totally vague. Before I write a new one, what does the current comment above mean? I don't want to have two different thoughts string together as one.